source/beamline/BioXAS/BioXASMainBeamline.h

```diff
@@ -47,7 +47,7 @@ class BioXASMainBeamline : public BioXASBeamline
 	{
 		if(!instance_){
 			instance_ = new BioXASMainBeamline();
-			instance_->initializeBeamlineSupport();
+			instance_->initializeBeamline();
```

Comment:
What is the reason for the name change? To me, 'initializing' the beamline sounds like what happens when we create all the beamline components.